server/athenian/api/controllers/jira_controller/get_jira_issues.py

```diff
@@ -124,6 +131,29 @@ async def _fetch_prs(
     return prs
 
 
+async def _fetch_comments(
+    issues: md.DataFrame,
+    account_info: AccountInfo,
+    mdb: Database,
+) -> dict[str, list[WebJIRAComment]]:
+    issue_ids = issues.index.values
+    extra_columns = [Comment.author_display_name, Comment.created, Comment.body]
+    jira_acc_id = account_info.jira_conf.acc_id
+    comments_map = await fetch_issues_comments(issue_ids, jira_acc_id, mdb, extra_columns)
+    return {
+        k: [_build_comment_web_model(comment, mdb) for comment in issue_comments]
+        for k, issue_comments in comments_map.items()
+    }
+
+
+def _build_comment_web_model(comment: Row, db: Database) -> WebJIRAComment:
+    return WebJIRAComment(
+        author=comment[Comment.author_display_name.name],
+        created=ensure_db_datetime_tz(comment[Comment.created.name], db),
```

Comment:
Note: we can put `np.datetime64` directly here, the serializers are ready for that.